docs/cmd/kn_service_create.md

```diff
@@ -88,7 +88,7 @@ kn service create NAME --image IMAGE
       --requests-cpu string               DEPRECATED: please use --request instead. The requested CPU (e.g., 250m).
       --requests-memory string            DEPRECATED: please use --request instead. The requested memory (e.g., 64Mi).
       --revision-name string              The revision name to set. Must start with the service name and a dash as a prefix. Empty revision name will result in the server generating a name for the revision. Accepts golang templates, allowing {{.Service}} for the service name, {{.Generation}} for the generation, and {{.Random [n]}} for n random consonants. (default "{{.Service}}-{{.Random 5}}-{{.Generation}}")
-      --scale int                         Minimum and maximum number of replicas.
+      --scale string                      Set the Minimum and Maximum number of replicas. You can use this flag to set both to a single value, or set a range with min/max values, or set either min or max values without specifying the other. Example: --scale 5 or --scale 1..5 or --scale 1.. or --scale ..5. (default "1")
```

Comment:
I would not use a “.” in the last example “or —scale ..5” since it’s confusing as to whether that last character is part of the syntax

Reply:
with each example listed we can add what does that pattern represent in `()` ? like `'--scale 1..' (min-scale=1, max scale undefined)` and also single quote the example themselves

Reply:
Sounds good. I'll make this update - have been working on breaking up the helper function at the moment. This will get added to my next commit. I've also updated the formatting so that it looks like other examples in the configuration_edit_flags.go file.

Reply:
+1 for putting literal examples in quotes, but i would suggest double quotes (") as in the example for `--revision-name` (just to be consistent).